### ISSUE TITLE
[Chip] Chip is unable to stretch its text vertically(like button), when it's constrained horizontally

### ISSUE THREAD

**Issue by ViktorScaletta** [material-components/material-components-android#2473]
I don't know if it's an intended behaviour, but it doesn't look like that. Maybe you can allow Chip to behave like Button in this case?

**Comment by drchen** [comment]
Can you elaborate? Wdym by "stretch"? Changing text size? Or changing chip size?

**Comment by ViktorScaletta** [comment]
@drchen I mean chip size. Button can stretch itself vertically according to containing text, if it's constrained horizontally. Chip can'
![Screenshot_5](https://user-images.githubusercontent.com/37725444/142968117-6645e37e-cb13-47ce-8407-50cfc45aa281.png)
t.

**Comment by anujpant08** [comment]
Is there any condition provided to have single line text or any size given to the button ?

**Comment by drchen** [comment]
@anujpant08 can you elaborate?

@ViktorScaletta I believe that's working as designed due to the nature of chips - chips are usually being used in a group, if we can't make chips' height fixed, it will be very hard to layout multiple chips.

**Comment by ViktorScaletta** [comment]
@drchen yeah, that's fair enough, but is that hard to implement some attribute, that could change its behaviour, like "app:canStretch" or whatever else?

**Comment by drchen** [comment]
Hmm. Let me move this to a feature request. We need some investigations to decide the efforts this requires. : )
